### PR TITLE
Add timer event.

### PR DESCRIPTION
**Pull request by XDLoLiK**

### CODE REVIEW

File: tools.hpp [hunk 1]
```diff
@@ -87,6 +89,11 @@ namespace booba { // boot of outstanding best api
         int32_t x, y; 
     };
 
+    struct TimerEventData
+    {
+        uint64_t time;
```
Comment (mishaglik) [inline tools.hpp:94]: What is time?

**Review by mishaglik — CHANGES_REQUESTED** [review]
Time is not specified.

File: tools.hpp (hunk 1 above)
Comment (mishaglik) [reply tools.hpp:97]: Ok

**Review by mishaglik — APPROVED** [review]
LGTM

**Comment by mishaglik** [comment]
Thank for contributing in our repository.

**Comment by Lord-KA** [comment]
Maybe [squash](https://stackoverflow.com/questions/5189560/how-do-i-squash-my-last-n-commits-together) commits next time?

**Pull request merged**